docs/environments/icp.md

```diff
@@ -36,7 +36,7 @@ For the second cluster, set the following value in `cluster/config.yaml` as foll
 cluster_name: cluster2
 ```
 
-## Post Install Configuration
+## Pre Install Configuration
```

Comment:
Here should be `post` I think as those operations are after ICP install finished.

Reply:
No, the namespace should be created AND CONFIGURED, before helm installation. If the namespace is created by the helm, it will have the default PSP. It can be configure later, but it is more complicated process. 
The `clusterimagepolicies` can be edited before or after helm installation, but better to do it before, so the Pod creation will not fail. Otherwise, it will fail and will be restarted after `clusterimagepolicies` editing.

Reply:
I see, please run `doctoc` to update ToC of this file.

Reply:
To prevent ambiguity, renamed the title to "Pre KubeFed Install Configuration", `ran doctoc`